src/instruments/instrumentstypes.h

```diff
@@ -171,6 +180,37 @@ struct PartInstrument {
 
 using PartInstrumentList = QList<PartInstrument>;
 
+struct ScoreOrderGroup {
+    int index;
+    QString family;
+    QString section { QString() };
+    QString unsorted { QString() };
+    bool bracket = false;
+    bool showSystemMarkings = false;
+    bool barLineSpan = false;
+    bool thinBracket = false;
+};
+
+struct InstrumentOverwrite {
```

Comment:
Seems that you can just make an alias instead of copying the whole structure

using InstrumentOverwrite = Ms::InstrumentOverwrite;

Reply:
Good catch, I wasn't even aware this was possible.
Done.